setup.py

```diff
@@ -142,7 +142,7 @@
         'SLIST = obspy.io.ascii.core',
         'PICKLE = obspy.core.stream',
         'CSS = obspy.io.css.core',
-        'DATAMARK = obspy.io.datamark.core',
+        'WIN = obspy.io.win.core',
```

Comment:
And also the other plugin lines.. yeah.. not sure about any negative side effects, but probably it should be OK like this..